packages/suite/src/views/wallet/send/Options/BitcoinOptions/CoinControl/UtxoSelectionList/UtxoSelection/UtxoSelection.tsx

```diff
@@ -120,6 +120,36 @@ const StyledFiatValue = styled(FiatValue)`
     ${typography.hint}
 `;
 
+type ResolveUtxoSpendableProps = {
+    utxo: AccountUtxo;
+    coinjoinRegisteredUtxos: AccountUtxo[];
+};
+
+// Same as MINIMAL_COINBASE_CONFIRMATIONS in '@trezor/utxo-lib'; It is redeclared here to avoid
+// some magic import/export errors. This is very niche stuff and probably never changes.
+// Also, this most probably bothers only developers on Regtest.
+const MINIMAL_COINBASE_CONFIRMATIONS = 100;
+
+const resolveUtxoSpendable = ({
+    utxo,
+    coinjoinRegisteredUtxos,
+}: ResolveUtxoSpendableProps): ReactNode | null => {
```

Comment:
as said below, this is an `FC`. So no need for return type `ReactNode | null` (btw null is assignable to ReactNode, so that can be simplified to `ReactNode`).

Reply:
After discussion, this is fine, just unclear naming.